alicloud/resource_alicloud_cen_instance.go

```diff
@@ -67,27 +67,25 @@ func resourceAlicloudCenInstanceCreate(d *schema.ResourceData, meta interface{})
 	request.Description = d.Get("description").(string)
 	request.ClientToken = buildClientToken(request.GetActionName())
 
-	var cen *cbn.CreateCenResponse
+	var response *cbn.CreateCenResponse
 	err := resource.Retry(5*time.Minute, func() *resource.RetryError {
-		req := *request
```

Comment:
Why remove the line 72?

Reply:
Maybe we don't need second level access address.